Sources/Remote Logging/Crash Logging/CrashLogging.swift

```diff
@@ -23,6 +23,9 @@ public class CrashLogging {
     /// crash log events will only be sent in Release builds.
     public static let forceCrashLoggingKey = "force-crash-logging"
 
+    // TODO: This could be made a configurable instance property, with default value in the `init`
+    public static let eventsFlushingTimeout: TimeInterval = 15
```

Comment:
Waiting to act upon this till we decided whether to go down the Sentry `flush` route.

Reply:
Any plan for addressing this TODO now that this PR is no longer draft?